src/Symfony/Component/Validator/Constraints/Locale.php

```diff
@@ -29,4 +29,13 @@ class Locale extends Constraint
 
     public $message = 'This value is not a valid locale.';
     public $canonicalize = false;
+
+    public function __construct($options = null)
+    {
+        if (!($options['canonicalize'] ?? false)) {
+            @trigger_error('The "canonicalize" option with value `false` is deprecated since Symfony 4.1 and will be removed in 5.0. Set it to `true` instead.', E_USER_DEPRECATED);
```

Comment:
we just merged a PR that removes all occurences of "and will be removed in 5.0."
would you mind removing them from this PR please?

Reply:
Sure! Done ;)